### DISCUSSION TITLE
How to configure multiple UEs to a single GnB using PacketRusher?

### DISCUSSION THREAD

**Discussion by madhavEdgeq** [HewlettPackard/PacketRusher#123] [Q&A] [answered]
I'm trying to simulate multiple UE's connected to a single GnB with traffic running. Is this possible to do with packet rusher? When I try running without the dedicated GnB: `sudo ./packetrusher multi-ue -n 6` I see six UE's created with a single GnB on the core but no interface is created so am not able to run traffic using iperf3. Is there a way to run traffic in this manner? Thanks!

**Comment by linouxis9** [comment, marked as answer]
Hi @madhavEdgeq,

Happy to see you using PacketRusher! To run multiples UEs at the same time, you need to add the --tunnel and --dedicatedGnb option: `sudo ./packetrusher multi-ue -n 6 --tunnel --dedicatedGnb`, which will create one gNodeB per UE and the associated tunnel.
For now, sadly, you cannot create multi-ue on a single gNodeB if you want to have a tunnel.
More information on --dedicatedGnb here: https://github.com/HewlettPackard/PacketRusher/issues/65#issuecomment-1895554491 and on why this limitation exists for now here: https://github.com/HewlettPackard/PacketRusher/issues/111#issuecomment-2049126443

Cheers,
Valentin

**Reply by madhavEdgeq** [reply]
Thanks for the quick response!
